src/main/java/be/kakumi/kachat/listeners/SendMessageListener.java

```diff
@@ -27,8 +27,8 @@ public void onSendMessage(AsyncPlayerChatEvent event) {
         event.setCancelled(true);
 
         Player player = event.getPlayer();
-        Channel channel = KAChatAPI.getInstance().getPlayerChannel(player);
         String message = event.getMessage();
+        Channel channel = getChannel(player, message);
```

Comment:
The override symbol, if any, should be removed from the message before starting all the formatting process.